frontend/src/routes/Applications/Overview.tsx

```diff
@@ -1129,6 +1129,7 @@ export default function ApplicationsOverview() {
         keyFn={keyFn}
         items={tableItems}
         filters={filters}
+        secondaryFilterIds={['cluster']}
```

Comment:
Commented elsewhere, but I think this should still be conditional on there being a large number of clusters.